iroh/src/endpoint.rs

```diff
@@ -634,68 +668,42 @@ impl Endpoint {
                 )
             })?;
 
-        debug!(
-            "connecting to {}: (via {} - {:?})",
-            node_id, addr, direct_addresses
-        );
+        debug!("connecting via {addr} - {direct_addresses:?}",);
```

Comment:
This message has always bothered me.  It is not clear to the user that `addr` in an internal QuicMappedAddr.  It also is not clear that a relay URL is used as well.  I think this is just a very very early debugging message from a different era.

Maybe something like `debug!(mapped_addr=?addr, relay_url=?node_addr.relay_url, ?direct_addresses, "Attempting connection...");`

If you feel like arguing to remove it entirely I probably won't fight you.